sql/core/src/test/resources/sql-tests/results/timestampNTZ/timestamp-ansi.sql.out

```diff
@@ -722,7 +722,6 @@ Unsupported type: timestamp_ntz
 -- !query
 select from_csv('26/October/2015', 't Timestamp', map('timestampFormat', 'dd/MMMMM/yyyy'))
 -- !query schema
-struct<>
+struct<from_csv(26/October/2015):struct<t:timestamp_ntz>>
 -- !query output
-java.lang.Exception
-Unsupported type: timestamp_ntz
+{"t":null}
```

Comment:
Why null? Which issue does the input have? No time part?

Reply:
ok. I have figured out: https://github.com/apache/spark/pull/28684/files#diff-d95bef2a8044eee384b21ba62b37d726c83eb7f48e768279db898197af3f5dd4R233

Reply:
Sorry, I should note about null here. Yes, `MMMMM` is an invalid format and I think it's intentional.